README.md

```diff
@@ -17,8 +17,10 @@ The @backtrace/browser is suitable for all JavaScript frameworks. Additional int
 of the capabilities of different frameworks.
 
 -   [@backtrace/browser](https://github.com/backtrace-labs/backtrace-javascript/tree/dev/packages/browser)
--   [@backtrace/node](https://github.com/backtrace-labs/backtrace-javascript/tree/dev/packages/browser)
--   [@backtrace/react](https://github.com/backtrace-labs/backtrace-javascript/tree/dev/packages/browser)
+-   [@backtrace/react](https://github.com/backtrace-labs/backtrace-javascript/tree/dev/packages/react)
```

Comment:
I think we should provide a link to the main branch instead

Reply:
Yes, thank you!